src/view/components/Drawer/index.tsx

```diff
@@ -74,7 +88,7 @@ class Drawer extends React.Component<Props, OwnState> {
         open={this.props.open}
         onClose={this.props.closeDrawer}
         onOpen={this.openDrawer}
-        content={<DrawerContents onPress={this.onPress} onLogout={this.onLogout} />}
+        content={this.createDrawerContents()}
```

Comment:
not a big deal here, but at some point when we want to do optimization, this can be cached so this it doesn't have to create a new one every time I think

Reply:
true, we just have to be really careful doing something like this in places where the props could change. In this case it should be safe because the contents are connected to redux I think?